### PR TITLE
Editor: Display Scene Camera on Main Scene

### LINKED ISSUE #15973: Editor: Looking through cameras
Hello,

It would be useful to have an option to look at the scene through PerspectiveCamera in Three.js Editor.

Here's Blender way to do it:

Camera

##### Three.js version

- [ ] Dev
- [x] r102
- [ ] ...

##### Browser

- [x] All of them
- [ ] Chrome
- [ ] Firefox
- [ ] Internet Explorer

##### OS

- [x] All of them
- [ ] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

### PR DESCRIPTION
**Pull request by Temdog007**
Fixes #15973

camera view 3

### CODE REVIEW

**Comment by Temdog007** [comment]
For testing: https://raw.githack.com/Temdog007/three.js/feature/CameraSwitch/editor/index.html

**Comment by looeee** [comment]
I'm getting "Unknown scene view type: undefined" errors when I tick the "show view" checkbox.

**Comment by Temdog007** [comment]
Remove cookies from raw.githack.com in order to get the most up-to-date version.

**Comment by looeee** [comment]
That's still not working for me I'm afraid. I can see the menu in the viewport now, but when I select the perspective camera nothing happens (no error messages either).

**Comment by looeee** [comment]
Oh, wait. I see that you've added a second smaller viewport. I think it would be better if the main viewport just shows the view from whatever camera is selected. Aside from that, the small viewport now covers the dropdown menus so I can't add anything to the scene once I've added a camera.

**Comment by Temdog007** [comment]
The dropdown menu can be moved in the settings tab.

Also just to note, the editor controls do not work when a scene camera is selected. I'll implement that in a future PR when this one gets merged.

**Comment by looeee** [comment]
Much better. Just two minor issues now:

1. The camera menu is still displaying over the drop down menus. I can't add anything to the scene
2. I'd rename "Scene Cameras" to "Camera Select Menu" in the settings. I'm not sure it needs to be there at all though

**Comment by Mugen87** [comment]
Please put the camera selection menu on the top right per default (as requested by @mrdoob here https://github.com/mrdoob/three.js/pull/15983#issuecomment-473416010).

File: editor/js/Strings.js [hunk 1]
```diff
@@ -244,6 +244,9 @@ var Strings = function ( config ) {
 			'sidebar/settings/theme': 'Theme',
 			'sidebar/settings/theme/light': 'light',
 			'sidebar/settings/theme/dark': 'dark',
+			'sidebar/settings/sceneCameras': 'Camera Select Menu',
+			'sidebar/settings/showSceneCameras': 'Show Camera Select Menu',
+			'sidebar/settings/showHelpers': 'Show Heleprs',
```
Comment (Mugen87) [inline editor/js/Strings.js]: Typo `Heleprs`.

**Review by Mugen87 — APPROVED** [review]

**Comment by Mugen87** [comment]
@Temdog007 Well done! 😉

**Review by looeee — APPROVED** [review]

**Comment by Temdog007** [comment]
Removed menu UI z-index fix from this PR and moved it to #16010

**Comment by mrdoob** [comment]
Still a bit too complicated but I'll simplify it post-merge.

**Comment by mrdoob** [comment]
Thanks!

**Comment by mrdoob** [comment]
@Temdog007 FYI ec73e0cfdeb1aad62269658fc70494ec07e11c6c

**Pull request merged**